packages/lab/src/Table/stories/TableColumnRenderers.stories.mdx

```diff
@@ -0,0 +1,202 @@
+import { Meta, Story, Preview } from "@storybook/addon-docs";
+
+import LinkTo from "@storybook/addon-links/react";
+
+import { Usage } from "storybook-root/blocks/Usage";
+
+import { HvTable } from "../..";
+
+import { 
+  AllColumnRenderers, 
+  TextColumnRenderer, 
+  NumberColumnRenderer, 
+  DateColumnRenderer,
+  ExpandColumnRenderer,
+  SwitchColumnRenderer,
+  TagColumnRenderer,
+  ProgressColumnRenderer,
+  DropdownColumnRenderer,
+} from "./TableColumnRenderers.stories.js";
+
+<Meta title="Lab/Table Column Renderers" />
+
+# Table column renderers
+
+The UI Kit library provides a collection of functions that work as **renderers** that ease the generation of various common column configurations.
+
+## `hv*Column` functions
+
+The `hv*Column` renderers are a set of functions that receive a series of parameters and outputs a `json` with the configuration required to achieve the desired look in that column
+
+you can use the provided renderers by importing them directly from the lab like this: 
+
+<Usage>{'import { hvTextColumn } from "@hitachivantara/uikit-react-lab";'}</Usage>
+
+The UI Kit encourages the using of these renderers because: 
+
+- Ensures data fallbacks in case of missing values.
+- Includes an overflow tooltip in case of long values.
+- Handles embedded component cases.
```

Comment:
I think this can be simplified.

```suggestion
# Table column renderers

The UI Kit library provides a collection of utility functions that, together with the <LinkTo kind="Lab/Table Hooks" story="Main" className="sbdocs-a">Table Hooks</LinkTo>, ease the setup of common column configurations, including the render function, alignment, missing data fallbacks and cell overflow.

They can be imported like this:
<Usage>{'import { hvTextColumn } from "@hitachivantara/uikit-react-lab";'}</Usage>
```